mix.exs

```diff
@@ -22,8 +22,8 @@ defmodule Astarte.Core.Mixfile do
   def project do
     [
       app: :astarte_core,
-      version: "1.1.0-dev",
-      elixir: "~> 1.11",
+      version: "1.1.0-alpha.0",
```

Comment:
Just double check: do we usually start with alpha.0 or alpha.1?

Reply:
We have a mix of both. I'd say we start _starting_ with 0 from now on.